src/ir/typescript_types.py

```diff
@@ -0,0 +1,201 @@
+from src.ir.types import Builtin
+
+
+import src.ir.builtins as bt
+import src.ir.types as tp
+
+class TypeScriptBuiltinFactory(bt.BuiltinFactory):
+    def get_language(self):
+        return "typescript"
+    
```

Comment:
This line here have 4 whitespaces. Ensure that there no whitespaces in this line; just \n.